### ISSUE TITLE
Adding win-win trades

### ISSUE THREAD

**Issue by rookbreezy** [Mhmdrza/free-market#1]
Really cool project! Love watching the trades fly by.

In the current simulation, it always ends in collapse because every trade is win-lose. I wonder how it can look if some trades are win-win, as many are in the real world.

**Comment by Mhmdrza** [comment]
Hello dear, sounds cool, what do you propose? what should happen per deal?
consider adding money out of no where is equal to printing money in real life; which means hyper-inflation; 
take a look at logic.js and suggest your idea

**Comment by rookbreezy** [comment]
I agree it should be without inflationary money printing. Not sure how to program it without making it much more complex.

A real world example would be: A farmer buying a tractor to be more productive. The tractor maker gets money. Both increase their wealth.

**Comment by Mhmdrza** [comment]
How about adding an extra object that is can be exchanged for $1 in a deal? but it is finite; if some one reached $0 or have no objects they are out of economy
